target/scripts/build/packages.sh

```diff
@@ -111,7 +154,8 @@ function _install_dovecot() {
     _log 'trace' 'Using Dovecot community repository'
     curl https://repo.dovecot.org/DOVECOT-REPO-GPG | gpg --import
     gpg --export ED409DA1 > /etc/apt/trusted.gpg.d/dovecot.gpg
-    echo "deb https://repo.dovecot.org/ce-2.3-latest/debian/bullseye bullseye main" > /etc/apt/sources.list.d/dovecot.list
+    # VERSION_CODENAME sourced from /etc/os-release
+    echo "deb https://repo.dovecot.org/ce-2.3-latest/debian/${VERSION_CODENAME} ${VERSION_CODENAME} main" > /etc/apt/sources.list.d/dovecot.list
```

Comment:
Just to clarify, this won't actually work anymore right? As Dovecot still does not provide a Debian Bookworm repo AFAIK?

I assume the next release is 2.4.0 / 3.0 from what I can tell. Perhaps they're trying to get that out the door first. Especially for a 3.x release that would probably make sense that they want to avoid the 2.x => 3.x update being implicit (_so prior Debian release repos would be stuck on the 2.x series_).

Reply:
As the Dovecot devs still have not bothered to update their community repo to include Debian 12, no, this won't work as of now. Personally I think this is a huge pain point; I guess it is what it is.

Reply:
Could try poking them on the email list? https://dovecot.org/mailman3/archives/list/dovecot@dovecot.org/

Reply:
That would be an option @paschun, thank you for noting this :) But truth be told, I do not have the time to do this, especially when considering that we'd then need to do this for all dependencies (to remain fair). Moreover, I'd need to clarify their ludicrous decision with 2.4 for community and 3 for "pro" first... In the meantime, staying with the Debian upstream is not too bad :)